src/fragments/lib-v1/auth/ios/getting_started/50_configureBackend.mdx

```diff
@@ -0,0 +1,200 @@
+<BlockSwitcher>
+<Block name="Amplify CLI (Create)">
+
+> Prerequisites: [Install and configure](/gen1/prev/[platform]/start/project-setup/prerequisites/#install-and-configure-the-amplify-cli) the Amplify CLI in addition to the Amplify libraries and [necessary dependencies](/gen1/prev/[platform]/build-a-backend/auth/set-up-auth/#install-amplify-libraries).
+
+To start provisioning auth resources in the backend, go to your project directory and **execute the command**:
+
+```bash
+amplify add auth
+```
+
+Enter the following when prompted:
+
+```console
+? Do you want to use the default authentication and security configuration?
+    `Default configuration`
+? How do you want users to be able to sign in?
+    `Username`
+? Do you want to configure advanced settings?
+    `No, I am done.`
+```
+
+> If you have previously enabled an Amplify category that uses Auth behind the scenes (e.g. API category), you can run the `amplify update auth` command to edit your configuration if needed.
+
+To push your changes to the cloud, **execute the command**:
+
+```bash
+amplify push
+```
+
+import ios3 from '/src/fragments/lib/auth/ios/getting_started/12_amplifyConfig.mdx';
+
+<Fragments fragments={{ swift: ios3 }} />
+
+import android4 from '/src/fragments/lib/auth/android/getting_started/12_amplifyConfig.mdx';
+
+<Fragments fragments={{ android: android4 }} />
+
+import flutter5 from '/src/fragments/lib/auth/flutter/getting_started/12_amplifyConfig.mdx';
+
+<Fragments fragments={{ flutter: flutter5 }} />
+
+</Block>
+
+<Block name="Amplify CLI (Import)">
```

Comment:
It was like that in the `lib` pages, I just copied them over to `lib-v1` since there were imports in `lib-v1` that were references these same pages but in `lib`